src/list_structure.js

```diff
@@ -49,3 +49,17 @@ function addTodo(todo) {
     }
 }
 
+function updateLS() {
+    todosEl = document.querySelectorAll('li')
+
+    const todos = []
+
+    todosEl.forEach(todoEl => {
+        todos.push({
+            text: todoEl.innerText,
+            completed: todoEl.classList.contains('completed')
+        })
+    })
+
+    localStorage.setItem('todos', JSON.stringify(todos))
```

Comment:
- [ ] Please make sure that your style matches the given design and also add the `Clear all completed` button.

### Your design
![image](https://user-images.githubusercontent.com/83803414/206872153-78bb5381-34d9-4468-8df1-64caa85c41cc.png)

### Given design
![image](https://user-images.githubusercontent.com/83803414/206872177-a0237b2f-b1a3-4fb8-a14d-de50e4ba8baa.png)